src/composables/useTour.ts

```diff
@@ -0,0 +1,384 @@
+import { Transaction, useTransactionsStore } from '@/stores/Transactions';
+import { SetupContext } from '@vue/composition-api';
+import { useAddressStore } from '../stores/Address';
+
+type OnboardingTourPages = '/' | '/transactions' | '/?sidebar=true'
+
+enum MobileOnboardingTourStep {
+    FIRST_ADDRESS,
+    TRANSACTIONS_LIST,
+    FIRST_TRANSACTION,
+    BITCOIN_ADDRESS,
+    WALLET_BALANCE,
+    RECOVERY_WORDS_ALERT,
+    MENU_ICON,
+    ACCOUNT_OPTIONS,
+    ONBOARDING_COMPLETED
+}
+
+type NetworkTourPages = '/network'
+enum NetworkTourStep {
+    TODO,
+}
+
+export type TourStepIndex = MobileOnboardingTourStep | NetworkTourStep;
+
+// eslint-disable-next-line max-len
+// https://github.com/floating-ui/floating-ui/blob/59d15d4f81a2d71d9b42d836e47d7114bc32f7f2/packages/core/src/types.ts#L4
+type Alignment = 'start' | 'end';
+type BasePlacement = 'top' | 'right' | 'bottom' | 'left';
+type AlignedPlacement = `${BasePlacement}-${Alignment}`;
+export type Placement = BasePlacement | AlignedPlacement;
+
+export interface TourStep {
+    page: OnboardingTourPages | NetworkTourPages;
+
+    // data for the steps of v-tour
+    tooltip: {
+        target: string,
+        content: string[],
+        params: Readonly<{
+            placement: BasePlacement | AlignedPlacement,
+        }>,
+    };
+
+    lifecycle?: {
+        prepareDOMPrevPage?: () => Promise<void>,
+        prepareDOMNextPage?: () => Promise<void>,
+        onBeforeMountedNextStep?: () => Promise<void>,
+        onBeforeMountedPrevStep?: () => Promise<void>,
+        onMountedStep?: (cbu: () => void) => void,
+    };
+
+    ui: {
+        // Elements that must have opacity to focus attention in other elements in screen
+        elementsWithOpacity?: string[], // array of selectors
```

Comment:
Not sure if `elementsWithOpacity` and `elementsWithoutInteractivity` are the best names. Suggestions for different names in all the PR are welcome! 🥲